src/build.cc

```diff
@@ -164,8 +164,26 @@ Edge* Plan::FindWork() {
   if (ready_.empty())
     return NULL;
 
+// TODO: jobserver client support for Windows
+#ifndef _WIN32
```

Comment:
It's never NULL right now, but actually, I can make it NULL on Windows in order to remove some of them...

Reply:
Ultimately, even if the pointer to the Jobserver object is NULL, compilation would still need a definition of the functions after the check for NULL pointer as if it would pass even though it never would, so I have to put the preprocessor macros back. Now we have even more...